guides/hack/03-expressions-and-operators/01-introduction.md

```diff
@@ -1,29 +1,43 @@
-**Topics covered in this section**
+When combined, operators evaluate according to their associativity. For more information, see [Operator Precedence](/hack/expressions-and-operators/operator-precedence).
 
-* [Arithmetic, `+`, `*` etc](addition.md)
-* [Assignment, `=`, `+=` etc](assignment.md)
-* [await](await.md)
-* [Bitwise Operators](bitwise-operators.md)
-* [Casting, `(int)`, `(float)`, `(bool)` `(string)`](casting.md)
-* [Coalesce, `??`](coalesce.md)
-* [Comparisons, `>` etc](comparisons.md)
-* [Decrement, `--`](incrementing-and-decrementing.md)
-* [echo intrinsic function](echo.md)
-* [Equality operators](equality.md)
-* [Error control, `@`](error-control.md)
-* [exit intrinsic function](exit.md)
-* [Increment, `++`](incrementing-and-decrementing.md)
-* [invariant intrinsic function](invariant.md)
-* [list intrinsic function](list.md)
-* [Logical operators, `&&`, `||`, `!`](logical-operators.md)
-* [Member selection, `->`, `?->`](member-selection.md)
-* [new](new.md)
-* [Operator precedence](operator-precedence.md)
-* [Pipe, `|>`](pipe.md)
-* [Scope resolution, `::`](scope-resolution.md)
-* [Spaceship, `<=>`](equality.md)
-* [String concatenation, `.`](string-concatenation.md)
-* [Subscripting, `[]`](subscript.md)
-* [Ternary, `?` `:`](ternary.md)
-* [Type Assertions `is`, `as`, `?as`](type-assertions.md)
-* [Yield](yield.md)
+## Assignment Operators
+* [Assignment](/hack/expressions-and-operators/assignment) (`=`, `+=`, and more)
+* [Bitwise](/hack/expressions-and-operators/bitwise-operators) (`&`, `|`, `^`, `<<`, `>>`, `~`)
+* [Coalesce](/hack/expressions-and-operators/coalesce) (`??`, `??=`)
+* [Pipe](/hack/expressions-and-operators/pipe) (`|>` then stored in `$$`)
+* [Spaceship](/hack/expressions-and-operators/equality#the-spaceship-operator) (`<=>`)
```

Comment:
Spaceship does not assign. It returns an integer where the sign represents the greater than / less than relationship between the LHS and the RHS.

Reply:
Moved. Thank you!